probes/contributorsFromOrgOrCompany/def.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: contributorsFromOrgOrCompany
+short: Checks whether a project has a contributions from Github users associated with a company or organization.
+motivation: >
+  
+implementation: >
+  This probe looks for the companies and organizations that contributors to the project belong to. 
```

Comment:
can we add a sentence about how we define a "contributor"? I think we use a minimum of 5 contributions to consider a user a contributor. There must also be a time window we use. Shall we add this here?